streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/controller/ThreadingController.java

```diff
@@ -65,15 +66,37 @@
      * The threading controller
      */
     public static ThreadingController getInstanceLowPriority() {
-        return INSTANCE_LOW_PRIORITY;
+        synchronized (ThreadingController.class) {
```

Comment:
I don't have a super strong preference either. I just like doing it there since n00bs might miss it if they aren't familiar with threading concepts.